test/e2e/upgrade/cidisruptiontester/cidisruptiontester.go

```diff
@@ -0,0 +1,77 @@
+package cidisruptiontester
+
+import (
+	"time"
+
+	"github.com/openshift/origin/pkg/monitor/backenddisruption"
+	"github.com/openshift/origin/test/extended/util/disruption"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/upgrades"
+)
+
+// ciDisruptionUpgradeTest tests the actual CI cluster where tests are running is able to reach an external service. This
+// is used to compare if we're actually observing disruption in the cluster under test, or if the CI cluster itself
+// is losing networking.
+//
+// The service in question is maintained by the TRT team, and running on the DPCR cluster in
+// the trt-monitoring namespace. (ci-disruption-tester svc)
+type ciDisruptionUpgradeTest struct {
+	backendDisruptionTest disruption.BackendDisruptionUpgradeTest
+}
+
+func NewCIDisruptionWithNewConnectionsTest() upgrades.Test {
+	ciDisruptTest := &ciDisruptionUpgradeTest{}
+	backend := backenddisruption.NewSimpleBackend(
+		"https://static.redhat.com/test/rhel-networkmanager.txt",
+		"ci-cluster-network-liveness",
+		"",
+		backenddisruption.NewConnectionType)
+	allowed := 1 * time.Second
+	ciDisruptTest.backendDisruptionTest =
+		disruption.NewBackendDisruptionTestWithFixedAllowedDisruption(
+			"[sig-trt] CI cluster remains able to communicate with an external service with new connections",
+			backend,
+			&allowed, // We'll let one second slide without reporting a problem
+			"CI cluster where tests are running may have network issues (not the cluster under test)",
```

Comment:
I'd add something that indicates this message came out of the test for New connections so we can distinguish it from messages that came out from the Used connections.

Reply:
The test name reflects new/reused, does that cover us?

Reply:
yes